Essentials/src/com/earth2me/essentials/commands/Commandptime.java

```diff
@@ -1,21 +1,20 @@
 package com.earth2me.essentials.commands;
 
-import com.google.common.collect.Lists;
-
 import com.earth2me.essentials.CommandSource;
+import com.earth2me.essentials.IUser;
 import com.earth2me.essentials.User;
 import com.earth2me.essentials.utils.DescParseTickFormat;
+import com.google.common.collect.Lists;
 import org.bukkit.Server;
 import org.bukkit.World;
-import org.bukkit.entity.Player;
 
 import java.util.*;
 
 import static com.earth2me.essentials.I18n.tl;
 
 
-public class Commandptime extends EssentialsCommand {
-    private static final Set<String> getAliases = new HashSet<>();
+public class Commandptime extends EssentialsLoopCommand {
+    protected static final Set<String> getAliases = new HashSet<>();
```

Comment:
This should remain private. Also, perhaps `<T> Arrays::asList(T...)` would be cleaner here instead of the set + static block.

Reply:
Is protected for its use in Commandpweather

Reply:
Only because you've forced it to be used in Commandpweather. Is there any particular reason why this is necessary? I think encapsulating each command is more important, and there is no other command as far as I know that borrows code from another directly like this.